src/api/ledger/parse/suspended-payment-finish.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+'use strict';
+const assert = require('assert');
+const sjclcodec = require('sjcl-codec');
+const utils = require('./utils');
+
+function convertHexToString(hexString) {
+  const bits = sjclcodec.hex.toBits(hexString);
+  return sjclcodec.utf8String.fromBits(bits);
+}
+
+function parsePayment(tx: Object): Object {
+  assert(tx.TransactionType === 'SuspendedPaymentFinish');
+
+  return utils.removeUndefined({
+    memos: utils.parsePaymentMemos(tx),
+    owner: tx.Owner,
+    offerSequence: tx.OfferSequence,
```

Comment:
Maybe, but note that one of the goals of this new API is that users aren't
supposed to be required to understand the rippled json. A lot of things
have been renamed already. Consider at some point documentation will be
written for this feature and @mduo13 or someone will have to spin
`offerSequence`. Ideally we'd change the name in rippled eh?

If you could name it anything, what would you name it?